tests/lora/utils.py

```diff
@@ -28,16 +28,16 @@ def init_random_lora(self,
             lora_alpha=1,
             lora_a=torch.rand([weight.shape[1], rank],
                               dtype=weight.dtype,
-                              device="cuda"),
+                              device="hpu"),
```

Comment:
We need to protect this in a way; it work with cuda also. You can refer to vllm/lora/model.py:96; in this branch